proto/vtrpc.proto

```diff
@@ -59,6 +106,6 @@ enum ErrorCode {
 // We use this so the clients don't have to parse the error messages,
 // but instead can depend on the value of the code.
 message RPCError {
```

Comment:
What other name will you use for the remote errors we send over rpc?
(just wondering if we need to rename this one too...)